app/routes/auth.route.ts

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import { changePasswordController, createNewUserController, loginUserController, resetPasswordController, resetPasswordRequestController, verifyUserController } from "../controllers/auth.controller.js";
```

Comment:
.js import keh

Reply:
I am sorry. I am a bit confused. Please are you saying I should not import with .js?